assets/scss/templates/_big-data.scss

```diff
@@ -93,4 +108,157 @@
       }
     }
   }
+
+  .dp-suggestion-bd-wrapper {
+    position: relative;
+    padding: $dp-spaces--lv5 $dp-spaces--lv2 $dp-spaces--lv2;
+    min-height: 300px;
+    margin-top: $dp-spaces--lv4;
+    border-radius: 3px;
+    border: 1px solid $dp-color-yellow;
+    background: $dp-color-white;
+    z-index: 1;
+
+    &:before {
+      content: '';
+      border: 1px solid $dp-color-yellow;
+      border-bottom-color: $dp-color-white;
+      border-right-color: $dp-color-white;
+      background: $dp-color-white;
+      position: absolute;
+      width: 22px;
+      height: 22px;
+      top: -11px;
+      right: 190px;
+      transform: rotate(45deg);
+    }
+
+    .dp-footer-panel {
+      position: relative;
+      display: flex;
+
+      span {
+        font-size: $dp-sizing--lvl2;
+      }
+    }
+  }
+
+  .bg-suggestion {
+    h4 {
+      text-transform: none;
+      border-bottom: 1px solid $dp-color-yellow;
+      margin-bottom: $dp-spaces--lv2;
+
+      span {
+        font-style: italic;
+        font-weight: normal;
+      }
+    }
+    span {
+      font-size: $dp-sizing--lvl2;
+    }
+
+    p {
+      font-size: $dp-sizing--lvl2;
+    }
+
+    .dp-tips-list {
+      li {
+        margin-bottom: $dp-spaces--lv1;
+        padding-left: $dp-spaces--lv5;
+        position: relative;
+        border-bottom: 1px solid $dp-color-snow;
+        padding-bottom: $dp-spaces--lv1;
+        line-height: 24px;
+
+        &:last-child {
+          border: none;
+        }
+      }
+
+      .advice-enabled {
+        &:before {
+          content: '';
+          position: absolute;
+          left: 0px;
+          top: 4px;
+          width: 20px;
+          height: 20px;
+          background: url('../img/advice-enabled.svg') no-repeat 0 0;
+          color: $dp-color-green;
+        }
+      }
+
+      .advice-disabled {
+        &:before {
+          content: '';
+          position: absolute;
+          left: 0px;
+          top: 4px;
+          width: 20px;
+          height: 20px;
+          background: url('../img/advice-disabled.svg') no-repeat 0 0;
```

Comment:
Why not add this svg as icon font?

Reply:
I wanted to manipulate the entire state with a single class. To use iconfont you had to add one more element and more classes.

Reply:
But to add this into doppler, we need to host the image or change the class. Can you add it as iconfont? If I'm not clear, let´s have a chat.

Reply:
@cbernat I did not understand this point well, this whole panel works under the dp-library class, and the images travel with the library. You have to use the library if or if to put it together. That means that the images are already in the library

Reply:
I've tested this and you are right! It works ok. Thanks for the clarification. If in the future we need this icons in another size perhaps it would be good to move this to icon font, but not now.